language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -157,29 +143,16 @@ public static void printSyntax(PrintStream out, List<Token> tokens) {
   /**
    * Connects to the Natural Language API using Application Default Credentials.
    */
-  public static CloudNaturalLanguage getLanguageService()
-    throws IOException, GeneralSecurityException {
-    GoogleCredential credential =
-        GoogleCredential.getApplicationDefault().createScoped(CloudNaturalLanguageScopes.all());
-    JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-    return new CloudNaturalLanguage.Builder(
-        GoogleNetHttpTransport.newTrustedTransport(),
-        jsonFactory, new HttpRequestInitializer() {
-              @Override
-              public void initialize(HttpRequest request) throws IOException {
-                credential.initialize(request);
-              }
-            })
-        .setApplicationName(APPLICATION_NAME)
-        .build();
+  public static LanguageServiceClient createLanguageService() throws IOException{
+	  return LanguageServiceClient.create();
```

Comment:
This seems a little pedantic, no? Might be clearer to just use `LanguageServiceClient.create()` directly and just delete this method.